compute_tools/src/compute.rs

```diff
@@ -2064,3 +2054,26 @@ pub fn forward_termination_signal() {
         kill(pg_pid, Signal::SIGINT).ok();
     }
 }
+
+// helper trait to call JoinSet::spawn_blocking(f), but propagates the current
+// tracing span to the thread.
+trait JoinSetExt<T> {
+    fn spawn_blocking_child<F>(&mut self, f: F) -> tokio::task::AbortHandle
+    where
+        F: FnOnce() -> T + Send + 'static,
+        T: Send;
+}
```

Comment:
Any reason to use a trait over a helper function?

Reply:
Seems a little nicer to write `joinset.spawn_blocking_child(...)` instead of `spawn_blocking_child(joinset, ...)`. That's all.